.github/workflows/continuous_integration.yaml

```diff
@@ -13,11 +13,14 @@ jobs:
           extra_nix_config: |
             keep-derivations = true
             keep-outputs = true
-      - run: nix develop --command mage
-        env:
-          CGO_ENABLED: 0
-          LDFLAGS: -w -s
-      - run: ./captain run captain-cli-ginkgo
+      - run: |
+          CGO_ENABLED=0 \
+            LDFLAGS="-w -s -X github.com/rwx-research/captain-cli.Version=testing-$(git rev-parse HEAD)" \
```

Comment:
💭  we use `git describe` (after a first tag is set) to make these versions more readable

Reply:
We don't always have a tag here- this is just the locally built version of the CLI, but Captain Cloud requires the versions set in the user agent to adhere to a certain format (or I wouldn't even bother setting it)

Reply:
`git describe` work without a tag set (as long as there's a tag in the git history)

Reply:
> annotated tag

I think this is the nuance I was missing, maybe

Reply:
yah `git tag  -a -m "" v1.2.3` or `git tag -s -m "" v1.2.3` (can include a message but 🤷‍♀️)